flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionJobVertex.java

```diff
@@ -126,19 +125,10 @@
 	private int maxParallelism;
 
 	/**
-	 * Serialized task information which is for all sub tasks the same. Thus, it avoids to
-	 * serialize the same information multiple times in order to create the
-	 * TaskDeploymentDescriptors.
+	 * Either store a serialized task information, which is for all sub tasks the same,
```

Comment:
Either the serialized task...... or the key of the offloaded task...?
`store` can be removed to keep same in (either or).